### ISSUE TITLE
Search sometimes returns the change description

### ISSUE THREAD

**Issue by hamishwillee** [diydrones/ardupilot-wiki-issue-tracker#117]
See http://plane.ardupilot.com/?s=RC1_REV

![image](https://cloud.githubusercontent.com/assets/5368500/9075289/9e10647e-3b56-11e5-8432-4fc9d28d3f1d.png)

**Comment by hamishwillee** [comment]
Further debug - if an edit reason is provided then this is found by search. If no edit reason provided (ie it is empty) then the data is searched. More debug required.

**Comment by hamishwillee** [comment]
The problem is that the "Edit reasons" field is just a rename of the **custom excerpt** field by the wiki plugin. Wordpress and our search solution uses this field for the excerpt if it has been defined. 

I have asked whether our search plugin can over-ride this behaviour. If not, we may need to modify or turn off revision information.

**Comment by hamishwillee** [comment]
This is still a problem. Tracked the problem in the plugin code to this line behaves differently if the default excerpt field has been set in the Wordpress UI or not. If it has been set in the UI, then no search is done in the post for matching text. If it has not been set then the method returns a string that contains a match:
```
               $excerpt = $default_excerpt = $searchwp_term_highlight->get_the_excerpt( $query );
```
Waiting on searchWP to confirm.

**Comment by hamishwillee** [comment]
Reproduced this problem on a bog-standard turnkey Wordpress VM. Have sent duplicator-copied site to searchwp to debug.

**Comment by hamishwillee** [comment]
Fixed in Term Highlight 2.0.5. What this means is that we can now use the excerpt for revision comments without worrying about breaking search.
